vault/token_store_test.go

```diff
@@ -2463,57 +2481,8 @@ func testTokenStore_HandleRequest_Lookup(t *testing.T, batch bool) {
 		"entity_id":        "",
 		"type":             tokenType,
 	}
-
```

Comment:
The diff makes this looks somewhat erroneous, but this deletion is basically to remove the "// Test via POST" step that seems to be happening twice. I suspect that it's a copy/paste duplication.